packages/renderer/src/lib/pod/PodCreateFromContainers.svelte

```diff
@@ -50,6 +50,7 @@ async function doCreatePodFromContainers() {
       return Object.entries(containerInspect.HostConfig.PortBindings).map(([key, value]) => {
         const container_port = parseInt(key.split('/')[0]);
         // we may not have any value
+        // eslint-disable-next-line no-null/no-null
         if (!value || value === null) {
```

Comment:
here you could directly delete `value === null` without adding the eslint rule. If it is null it gets in the if branch in any case because of `!value` clause

Reply:
I've reviewed all if and change them